POUFs/TAF-POUF/pouf2.md

```diff
@@ -26,27 +26,61 @@ for metadata generation, it implicitly depends on the reference implementation
 POUF’s protocols.
 
 Metadata and target files are stored in a git repository referred to as an
-authentication repository.  An authentication repository contains information
+authentication repository. An authentication repository contains information
 needed to securely clone and update other git repositories (referred to as
 target repositories), including their URLs and additional custom data that can
 be used by TAF's implementers. This is specified in special target files,
 `repositories.json` and `mirrors.json` - regular TUF  target files (whose
-lengths and hashes are stored in `targets.json` and are signed by the top-level
-targets role) which are of special importance to TAF.
-
-A client downloads an authentication repository and all of the referenced
-repositories by running TAF’s updater and specifying the authentication
-repository’s URL. Repositories are cloned and updated using git. The updater
-runs validation before permanently storing anything on the client’s machine:
+lengths and hashes are stored in `targets.json` and which are signed by the
+top-level targets role) which are of special importance to TAF.
+
+TAF focuses on protecting git repositories from unauthorized pushes. It is
+designed to record valid commits for target repositories in the authentication
+repository. If an attacker manages to compromise a user who has write access to
+protected repositories and creates new commits, TAF will detect this by
+comparing these new commits to a list of valid commits specified in the
+authentication repository. In order to register these new commits, the attacker
+has to modify TUF metadata files as well. Unless they also gain access to
+`targets` or `root` keys, they cannot do this without the attempt being
+detected by TAF. TAF's users are encouraged to store signing keys for the
+`target` and `root` roles offline on hardware tokens to ensure their safety.
+
+In essence, while TAF ensures the validity of commits in target repositories,
+it makes no claims about the integrity of their contents. TAF relies on Git's
+default artifact integrity protection. Git, however, still primarily relies on
+SHA-1, even though it has been proven that this hash function is vulnerable to
+collision attacks. If an attacker manages to gain access to the original target
+repositories, they could potentially exploit the SHA-1 weakness. As TAF stores
+verified URLs of its target repositories, users are protected from _mirrors_ of
+legitimate repositories presenting a colliding artifact in place of the
+original artifact.
+
+<!--
```

Comment:
I think this may be overkill. I took a pass at focusing more on TAF + Git rather than SHAttered. WDYT? @renatav